### ISSUE TITLE
Leo Leo. Reset stops working after a while.

### ISSUE THREAD

**Issue by andybutler** [musicdevghost/eurorack#17]
External reset stops working when a charge builds up on the side of C41 not connected to R106(point named VRESET).
So it works for a while, then stops. You fiddle around it works a bit then stops. With test meter on VRESET it works.

Solved. Add 2M resistor from VRESET to ground.

**Comment by cctvfm** [comment]
I noticed this issue on the VCLOCK input, which uses an identical circuit. Can you confirm the 2M to ground from the input side of C38 solved the issue long-term?

**Comment by andybutler** [comment]
Yep, it solves.Input to the op amp is high impedance and floating, so charge can build up. The resistor dissipates the charge to ground.Currently the only issue remaining with my unit is a random odd state on bootup. Usually need to run X an Y clocks for a few steps to clear.andyBOn 16 Jan 2025 18:12, cctvfm ***@***.***> wrote:
I noticed this issue on the VCLOCK input, which uses an identical circuit. Can you confirm the 2M to ground from the input side of C38 solved the issue long-term?
